scss/_card.scss

```diff
@@ -0,0 +1,92 @@
+.card{
+    width: 340px;
+    height: 355px;
+    position: relative;
+    border: 1px solid $light_grey;
+    list-style-type: none;
+    &__image{
+        height: 190px;
+        width: 330px;
```

Comment:
I would expect this width set to %, to make the blocks adapt to the width of the viewport